test/components/views/settings/tabs/user/AccountUserSettingsTab-test.tsx

```diff
@@ -170,7 +238,7 @@ describe("<GeneralUserSettingsTab />", () => {
             expect(within(screen.getByTestId("mx_AccountPhoneNumbers")).getByLabelText("Loading…")).toBeInTheDocument();
         });
 
-        it("should display 3pid email addresses and phone numbers", async () => {
+        it.skip("should display 3pid email addresses and phone numbers", async () => {
```

Comment:
Why is this skipped?

Reply:
oops, accidental from debugging tests - good spot, thanks.